packages/thailand_city_data/example/thailand_city_data_example.dart

```diff
@@ -0,0 +1,173 @@
+import 'dart:collection';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:characters/characters.dart';
+import 'package:excel/excel.dart';
+
+void main(List<String> args) async {
+  final list = await excelToJson();
+
+  final provinceMap = getProvince(list[1]);
+  await saveToFile(
+    provinceMap,
+    field: 'Map<String, String> thProvincesData',
+    fileName: 'province',
+  );
+
+  final districtMap = getDistrict(list[1], list[2]);
+  await saveToFile(
+    districtMap,
+    field: 'Map<String, dynamic> thCitiesData',
+    fieldGeneric: '<String, Map<String, Map<String, String>>>',
+    fileName: 'district',
+  );
+
+  await Process.run('dart', ['format', 'lib']);
+  await Process.run('dart', ['fix', '--apply']);
+}
+
+/// 保存到文件
+Future<void> saveToFile(
+  Object object, {
+  required String field,
+  String fieldGeneric = '',
+  required String fileName,
+}) async {
+  print('开始执行写入文件');
+  try {
+    final jsonFile = File('lib/src/$fileName.dart');
+
+    //判断文件是否存在
+    if (!await jsonFile.exists()) {
+      await jsonFile.create();
+      print('文件创建成功');
+    } else {
+      print('文件已存在');
+    }
+
+    //写入的数据
+    final contents = JsonEncoder.withIndent('  ').convert(object);
+
+    // json文件写入
+    await jsonFile.writeAsString(
+      '''/// Generate by xxx, DO NOT EDIT IT
+const $field = $fieldGeneric$contents;
+''',
+      flush: true,
+    );
+
+    print('执行写入文件成功');
+  } catch (e) {
+    print('执行写入文件失败');
+  }
+}
+
+/// 生成行政区划之县的json
+Map<String, Map<String, Map<String, String>>> getDistrict(
+  List<Map<String, String>> provinceList,
+  List<Map<String, String>> districtList,
+) {
+  final map =
+      SplayTreeMap<String, Map<String, Map<String, String>>>((key1, key2) {
+    return key1.compareTo(key2);
+  });
+
```

Comment:
如果是使用String默认的compareTo, 不需要额外传比较方法
```suggestion
final map = SplayTreeMap<String, Map<String, Map<String, String>>>();
```

Reply:
其他两个地方也一样

Reply:
已解决